src/test/java/com/battleship/backend/models/BoardTest.java

```diff
@@ -78,6 +80,30 @@ void testHasShip(){
         assertTrue(testBoard.hasShip(testShip));
     }
 
+    @Test
+    void testShipListEqualsReturnsTrueIfAllShipNamesMatch(){
+        ArrayList<Ship> testList = new ArrayList<Ship>();
+        testList.add(new Ship("test1", 2));
+        testList.add(new Ship("test2", 2));
+
+        testBoard.addShip(new Ship("test1", 2), 0 ,0);
+        testBoard.addShip(new Ship("test2", 2), 1 ,0);
+
+        assertTrue(testBoard.shipListEquals(testList));
+    }
+
+    @Test
+    void testShipListEqualsReturnsFalseIfNamesDoNotMatch(){
```

Comment:
Nitpick: can we update these two tests to either be:
`testShipListEqualsFalse`
or
`testShipListReturnsFalse`
vs
`testShipListEqualsReturns`

Reply:
The method name was shipListEquals, which led to that unfortunate name. I have changed the name of the method to isPermutationOfShipList, so the updated test names make more sense now.